src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -1,62 +1,156 @@
+import { ArtistInfoQuery } from "__generated__/ArtistInfoQuery.graphql"
+import { ContextConsumer } from "Artsy"
+import { Mediator } from "Artsy/SystemContext"
+import FollowArtistButton from "Components/FollowButton/FollowArtistButton"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import { EntityHeader } from "Styleguide/Components/EntityHeader"
+import { get } from "Utils/get"
 
 import {
-  ArtistBio,
-  MarketInsights,
-  MarketInsightsProps,
-  SelectedExhibitions,
-  SelectedExhibitionsProps,
+  ArtistBioFragmentContainer as ArtistBio,
+  MarketInsightsFragmentContainer as MarketInsights,
+  SelectedExhibitionFragmentContainer as SelectedExhibitions,
 } from "Styleguide/Components"
-import { EntityHeader } from "Styleguide/Components/EntityHeader"
 
-interface ArtistInfoProps
-  extends MarketInsightsProps,
-    SelectedExhibitionsProps {
-  name: string
-  bio: string
-  credit?: string
+interface ArtistInfoProps {
+  artist: ArtistInfo_artist
+  user: User
+  mediator?: Mediator
 }
 
-export class ArtistInfo extends React.Component<ArtistInfoProps> {
-  render() {
-    // FIXME: Fetch from relay
-    const props = {
-      href: "/artist/francesca-dimattio",
-      imageUrl: "https://picsum.photos/110/110/?random",
-      name: "Francesca DiMattio",
-      meta: "American, b. 1979",
+export const ArtistInfo: SFC<ArtistInfoProps> = props => {
+  const imageUrl = get(props, p => p.artist.image.cropped.url)
+
+  return (
+    <>
+      <StackableBorderBox p={2} flexDirection="column">
+        <EntityHeader
+          name={props.artist.name}
+          meta={props.artist.formatted_nationality_and_birthday}
+          imageUrl={imageUrl}
+          FollowButton={
+            <FollowArtistButton
+              artist={props.artist}
+              user={props.user}
+              onOpenAuthModal={() => {
+                props.mediator.trigger("open:auth", {
+                  mode: "signup",
+                  copy: `Sign up to follow ${props.artist.name}`,
+                  signupIntent: "follow artist",
+                  afterSignUpAction: {
+                    kind: "artist",
+                    action: "follow",
+                    objectId: props.artist.id,
+                  },
+                })
+              }}
+              render={({ is_followed }) => {
+                return (
+                  <Sans
+                    size="2"
+                    weight="medium"
+                    style={{
+                      cursor: "pointer",
+                      textDecoration: "underline",
+                    }}
+                  >
+                    {is_followed ? "Following" : "Follow"}
+                  </Sans>
+                )
+              }}
+            >
+              Follow
+            </FollowArtistButton>
+          }
+        />
+        <Spacer mb={1} />
+        <ArtistBio bio={props.artist} maxChars={300} />
```

Comment:
We should double check with @design if this `300` is a valid collapse length. I copied this number to artwork details texts as well. Was wondering if that was from design or just somebody putting a random number.

Reply:
I just dropped the copy from Zeplin into a char counter and its roughly 300